scripts/analysisTools/w_mass_13TeV/run2Dsmoothing.py

```diff
@@ -690,22 +687,56 @@ def runSmoothing(inputfile, histname, outdir, step, args, effHist=None):
     #
     # sfFolder = data_dir + "/muonSF/"
     # effSmoothFile = f"{sfFolder}efficiencies3D_rebinUt2.pkl.lz4"
+
+    '''
     sfFolder = data_dir + "/muonSF/"
     sfFolderVtxAgn = data_dir + "/muonSF/intermediate_vtxAgnosticIso/"
     effSmoothFile = sfFolderVtxAgn + "efficiencies3D_rebinUt2_vtxAgnPfRelIso04.pkl.lz4"
-    #
+    '''
+
+    inputRootFile_2016PostVFP = {
+        "iso": data_dir + "/muonSF/iso3DSFVQTsingularity.root",
+        "isonotrig": data_dir + "/muonSF/intermediate_vtxAgnosticIso/isonotrig3DSFVQTsingularity.root",
+        "isoantitrig": data_dir + "/muonSF/intermediate_vtxAgnosticIso/isofailtrig3DSFVQTsingularity.root",
+        "triggerplus": data_dir + "/muonSF/triggerplus3DSFVQTextended.root",
+        "triggerminus": data_dir + "/muonSF/triggerminus3DSFVQTextended.root",
+    }
+
     inputRootFile = {
-        "iso": f"{sfFolderVtxAgn}iso3DSFVQTsingularity.root",
-        "isonotrig": f"{sfFolderVtxAgn}isonotrig3DSFVQTsingularity.root",
-        "isoantitrig": f"{sfFolderVtxAgn}isofailtrig3DSFVQTsingularity.root",
-        "triggerplus": f"{sfFolder}triggerplus3DSFVQTextended.root",
-        "triggerminus": f"{sfFolder}triggerminus3DSFVQTextended.root",
+        "iso": "/home/users/ruben/eff3D_Davide_2017/3dsfisolation.root",
+        "isonotrig": "/home/users/ruben/eff3D_Davide_2017/3dsfisonotrig.root",
+        "triggerplus": "/home/users/ruben/eff3D_Davide_2017/3dsftriggerplus.root",
+        "triggerminus": "/home/users/ruben/eff3D_Davide_2017/3dsftriggerminus.root",
     }
 
     parser = common_plot_parser()
+
     parser.add_argument(
-        "outdir", type=str, nargs=1, help="output directory to save things"
+        "-i",
+        "--indir",
+        help="Input directory where the 3d efficiencies measured on data are stored"
     )
+
+    parser.add_argument(
+        "-o",
```

Comment:
I think the outdoor is mandatory, so the original option is better. I also don't see that you are changing any line which uses it, so the risk is that you might call the script without using it and then the code crashes because None is not a valid value. If you always have to call an option, better to keep it a positional argument.